torchvision/prototype/transforms/_transform.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Dict, Optional
+
+from torch import nn
+from torchvision.prototype.utils._internal import apply_recursively
+
+from .functional._utils import Dispatcher
+
+
+class Transform(nn.Module):
```

Comment:
You need a constructor that calls the [logger](https://github.com/pytorch/vision/blob/main/torchvision/transforms/transforms.py#L89-L90). We should also consider on a separate PR to handle the logger on the functional/kernel level. Could be done on the decorator but that won't capture JIT.